src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function getSum(firstNum) {
+    return function getSum2(secNum) {
+      count++;
+
+      if (count >= 3 && (count % 2) === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return firstNum + secNum;
+      }
```

Comment:
```suggestion
      if (count >= 3 && (count % 2) === 0) {
        return 'Bzzz... Error!';
      }
      
       return firstNum + secNum;
      
```
else looks redundant here